pipeline/config-prod.groovy

```diff
@@ -341,33 +341,33 @@ environments {
             }
             resources {
                 node {
-                    cpu_request = "100m"
-                    cpu_limit = "150m"
-                    memory_request = "512Mi"
-                    memory_limit = "1Gi"
+                    cpu_request = "20m"
+                    cpu_limit = "100m"
+                    memory_request = "160Mi"
+                    memory_limit = "256Mi"
                     replica_min = 2
                     replica_max = 4
                 }
                 nginx {
-                    cpu_request = "100m"
-                    cpu_limit = "150m"
-                    memory_request = "256Mi"
-                    memory_limit = "512Mi"
-                    replica_min = 2
-                    replica_max = 4
+                    cpu_request = "10m"
+                    cpu_limit = "50m"
+                    memory_request = "96Mi"
+                    memory_limit = "160Mi"
+                    replica_min = 3
+                    replica_max = 6
                 }
                 python {
-                    cpu_request = "200m"
-                    cpu_limit = "400m"
-                    memory_request = "1.5Gi"
-                    memory_limit = "3Gi"
+                    cpu_request = "100m"
+                    cpu_limit = "200m"
+                    memory_request = "512Mi"
+                    memory_limit = "2Gi"
                     uwsgi_threads = 2
                     uwsgi_processes = 4
                     replica_min = 2
                     replica_max = 4
                 }
                 python_lite {
-                    cpu_request = "100m"
+                    cpu_request = "10m"
```

Comment:
This seems a little low compared to test, and doesn't have a cpu limit.

Reply:
The limit is right underneath it 😅 
But yeah, the python_lite pod is actually not using more than 0.001 CPU in general, so I reduced the CPU to reflect that.

Good catch about test tho, updated test/dev to reflect the same values